doc/examples/collector/src/main/scala/com/twitter/my-blocking-collector/MyBlockingCollectorService.scala

```diff
@@ -0,0 +1,30 @@
+package com.twitter.myblockingcollector
+
+import com.twitter.finagle.stats.StatsReceiver
+import com.twitter.server.TwitterServer
+import com.twitter.util.{Await, Future}
+import com.twitter.zipkin.cassandra.CassieSpanStoreFactory
+import com.twitter.zipkin.collector.{SpanReceiver, ZipkinQueuedCollectorFactory}
+import com.twitter.zipkin.thriftscala.Span
+import com.twitter.zipkin.receiver.scribe.ScribeSpanReceiverFactory
+import com.twitter.zipkin.storage.WriteSpanStore
+import com.twitter.zipkin.zookeeper.ZooKeeperClientFactory
+
+object ZipkinCollectorServer extends TwitterServer
```

Comment:
@jamescway @adriancole What was the verdict on builder vs factory? We use the factory method so if that's the case I can move it to the file you suggested.